.github/workflows/ci.yml

```diff
@@ -41,13 +41,10 @@ jobs:
       matrix:
         os:
           - 'debian-9'
-          - 'debian-9-chef-13'
           - 'debian-10'
           - 'centos-7'
-          - 'centos-7-chef-13'
           - 'centos-8'
```

Comment:
Do you also want stream?

Reply:
Probably wouldn't hurt to add it. I'll do that.

Reply:
I'll do that in a different PR. I want to just get this out in case I run into issues on stream.